src/i18n/en.toml

```diff
@@ -145,17 +145,22 @@ deployedAppVersion = "Deployed app version"
 [login.form]
 titleLogin = "Login"
 titlePasswordReset = "Forgotten password"
+titleResetFinished = "You will receive a password reset link in your email"
 descriptionPasswordReset = "We will email you a link to reset your password."
+descriptionResetFinished = "We have just emailed you a link to reset your password at anonym@dopracenakole.cz. If you can't find the link, check your spam folder."
```

Comment:
Perhaps this email address anonym@dopracenakole.cz should have a place in the global configuration file.

Reply:
Good point. Edited.